code/modules/surgery/organs/organ_external.dm

```diff
@@ -670,8 +670,11 @@ Note that amputating the affected organ does in fact remove the infection from t
 	//robot limbs take reduced damage
 	if(!make_tough)
 		brute_mod = 0.66
-		burn_mod = 0.66
+		burn_mod = 0.66		
+		if(owner && (ROBOTIC in owner.dna.species.species_traits))
+			return
```

Comment:
doing this will prevent IPCs limbs getting set to the right company sprite, and will mean that they will have bones, and that they will have bones that can break.

Did you test this?